src/main/java/mate/academy/bookstore/serivce/orderitem/OrderItemServiceImpl.java

```diff
@@ -0,0 +1,66 @@
+package mate.academy.bookstore.serivce.orderitem;
+
+import jakarta.transaction.Transactional;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import mate.academy.bookstore.dto.orderitem.OrderItemResponseDto;
+import mate.academy.bookstore.exception.EntityNotFoundException;
+import mate.academy.bookstore.mapper.OrderItemMapper;
+import mate.academy.bookstore.model.CartItem;
+import mate.academy.bookstore.model.Order;
+import mate.academy.bookstore.model.OrderItem;
+import mate.academy.bookstore.repository.OrderItemRepository;
+import mate.academy.bookstore.repository.OrderRepository;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class OrderItemServiceImpl implements OrderItemService {
+    private final OrderItemMapper orderItemMapper;
+    private final OrderItemRepository orderItemRepository;
+    private final OrderRepository orderRepository;
+
+    @Override
+    @Transactional
+    public Set<OrderItem> toOrderItem(Set<CartItem> cartItems) {
+        return cartItems.stream()
+                .map(orderItemMapper::toOrderItem)
+                .collect(Collectors.toSet());
+    }
```

Comment:
I do not think we need a service method for that, just use the orderItemMapper directly instead of this method